<launch>
  <node name="aruco_marker_calibrate" pkg="aruco_detect" type="aruco_calibrate.py" output="screen" >
    <param name="aruco_type" type="str" value="DICT_6X6_1000" />
    <param name="aruco_length" type="double" value="0.05" />
    <param name="aruco_update_rate" type="double" value="1" />
    <param name="aruco_save_dir" type="str" value="/home/kkw0418/urp_ws/src/ArUcoROSpy/src/" />
    <param name="aruco_main_marker_id" type="int" value="6" />

    <param name="camera_img_topic" type="str" value="/camera/color/image_raw"/>
    <param name="camera_info_topic" type="str" value="/camera/color/camera_infoo"/>
    <param name="camera_frame_id" type="str" value="/camera_link"/>
  </node> 
</launch>
